raft/server.cc

```diff
@@ -1431,7 +1442,7 @@ term_t server_impl::get_current_term() const {
 
 future<> server_impl::wait_for_apply(index_t idx, abort_source* as) {
     if (as && as->abort_requested()) {
-        throw request_aborted();
+        throw request_aborted(format("Aborted while waiting for applying entry: {}, last applied entry: {}", idx, _applied_idx));
```

Comment:
Similar to my previous comment in `wait_for_entry`, here I would use the word "before" compared to the message below which uses "while".